src/constants/constants.js

```diff
@@ -0,0 +1,7 @@
+module.exports = {
+  ACTION_TYPE: {
+    GET_NEWS_REQUESTED: 'NEWS_REQUESTED',
```

Comment:
To avoid further confusion rename actions to GET_NEWS_REQUESTED: 'GET_NEWS_REQUESTED' ....